src/java.base/share/classes/java/security/spec/PSSParameterSpec.java

```diff
@@ -157,13 +160,21 @@ public PSSParameterSpec(String mdName, String mgfName,
     /**
      * Creates a new {@code PSSParameterSpec}
      * using the specified salt length and other default values as
-     * defined in PKCS#1.
+     * defined in the PKCS #1 standard.
      *
-     * @param saltLen the length of salt in bytes to be used in PKCS#1
-     *         PSS encoding
+     * @param saltLen the length of salt in bytes
      * @throws    IllegalArgumentException if {@code saltLen} is
      *         less than 0
+     * @deprecated This constructor uses the default values defined in
+     *         the PKCS #1 standard except for the salt length. Some of these
+     *         defaults are no longer recommended due to advances in
+     *         cryptanalysis -- see the
+     *         <a href="https://tools.ietf.org/rfc/rfc8017.txt">PKCS#1 v2.2</a>
+     *         standard for more details. Thus, it is recommended to explicitly
+     *         specify all desired parameter values with
+     *         {@link #PSSParameterSpec(String, String, AlgorithmParameterSpec, int, int) PSSParameterSpec}.
```

Comment:
Same comment about seeing the full signature of the ctor as mentioned above.

Reply:
Yes.